### ISSUE TITLE
KeyErrors in config

### ISSUE THREAD

**Issue by oadams** [CoEDL/elpis#69]
On branch `master`, following the steps outlined [here](https://github.com/CoEDL/elpis/wiki/Elpis-workshop-online#about-the-steps)

```
File "/elpis/elpis/endpoints/dataset.py", line 14, in new
    dataset = kaldi.new_dataset(request.json['name'])
  File "/elpis/elpis/wrappers/objects/interface.py", line 91, in new_dataset
    existing_names = self.list_datasets()
  File "/elpis/elpis/wrappers/objects/interface.py", line 110, in list_datasets
    names = [name for name in self.config['datasets'].keys()]
  File "/elpis/elpis/wrappers/objects/fsobject.py", line 119, in __getitem__
    return self._load()[key]
KeyError: 'datasets'
```

Looks like this stems from `commit 5a07f93646ec2993a65ce3474c3f8598f03d87cb`.

Similar things happen with KeyError: models when creating a new model. I can work around this by manually editing `state/interface.json` but is there a better approach?

**Comment by benfoley** [comment]
The current way of initialising the state's config objects wipes any values of prior config objects because the keys are set to empty objects on init. This is problematic for using the CLI because we then can't seperate the training and inferring steps, as they both run the init method when creating their interfaces. I've updated the `interface.py` file to at least allow the GUI to not throw the KeyErrors in config. Will need to review the way that the interface is initialised. Leaving this open to remind me to come back to it...

**Comment by benfoley** [comment]
Should have been fixed with #88
